.scripts/run_config.sh

```diff
@@ -37,16 +37,25 @@ cleanup() {
 }
 trap cleanup EXIT
 
+mkdir -p "${root_path}/tmp"
+
 if [ $target == "Default" ]; then
 
-  PASSWORD=${DB_PASSWORD:-$CREDENTIALS}
+  storage_target=$(basename $(dirname "${model_path}"))
+  if [ "${storage_target}" = "redshift" ] ; then
+    PASSWORD=${REDSHIFT_PASSWORD:-$CREDENTIALS}
+  elif [ "${storage_target}" = "snowflake" ] ; then
+    PASSWORD=${SNOWFLAKE_PASSWORD:-$CREDENTIALS}
+  else
+    echo "Storage target mismatch!" >&2 ; exit 1;
+  fi
 
 elif [ "$target" == "BigQuery" ]; then
 
-  BIGQUERY_CREDS=${BIGQUERY_CREDS:-$CREDENTIALS}
+  PASSWORD=${BIGQUERY_CREDS:-$CREDENTIALS}
```

Comment:
Using PASSWORD here breaks the sed later on - because the value isn't compatible for a simple substitution. It throws:

`sed: 1: "s/PASSWORD_PLACEHOLDER/ ...": bad flag in substitute command: 'c'`

(It's also not actually a password. It's a JSON of service account credentials for BQ)

Since the BQ playbooks don't need passwords, it's fine not to have any value in PASSWORD since that `sed` is just substituting nothing.

Reply:
Ok, thank you for trying it with bigquery! I will correct this so.